presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -188,18 +205,18 @@ public StorageManagerConfig setRecoveryThreads(int recoveryThreads)
         return this;
     }
 
-    @Config("storage.max-compaction-threads")
```

Comment:
Why is this renamed? Compaction and organization are different operations and should be able to have different thread counts